desktop/extensions-sdk/extensions/publish.md

```diff
@@ -30,9 +33,9 @@ These guidelines don't replace Docker's terms of service or guarantee approval.
 - Test your extension on various platforms (Mac, Windows, Linux)
 - Read the [Terms of Service](https://www.docker.com/legal/extensions_marketplace_developer_agreement/){: target="_blank" rel="noopener" class="_" }
 
-### After you submit
+### After you submit requesting a review from Docker Extensions team
 
-Once you’ve submitted your extension, here is what you can expect from the review process:
+Once you’ve submitted your extensionand requested a review from the Docker Extension team, here is what you can expect from the process:
```

Comment:
```suggestion
Once you’ve submitted your extension and requested a review from the Docker Extension team, here is what you can expect from the process:
```

Reply:
I'd change the title After you submit - Extensions being reviewed by the Docker Extensions team

Reply:
I have done things in another way in 82ee3e80ab